### PR TITLE
Make title and categories be within the apps themselves

### PR DESCRIPTION
**Pull request by SimonBiggs**

### CODE REVIEW

**Review by sjswerdloff — APPROVED** [review]
Are CATEGORY and TITLE special variables known to streamlit?
It isn't clear how these are getting used (I didn't see them in the files with calls to st.title(...))

File: lib/pymedphys/_streamlit/index.py [hunk 1]
```diff
@@ -170,14 +138,14 @@ def index():
         applications_in_this_category = [
             item
             for item in APPLICATION_OPTIONS.items()
-            if item[1]["category"] == category_key
+            if item[1].CATEGORY == category_key
```
Comment (SimonBiggs) [inline lib/pymedphys/_streamlit/index.py:141]: Here

File: lib/pymedphys/_streamlit/index.py [hunk 2]
```diff
@@ -170,14 +138,14 @@ def index():
...
         ]
 
         if not applications_in_this_category:
             st.write("> *No applications are currently in this category.*")
 
         for app_key, application in applications_in_this_category:
-            if st.button(application["label"]):
+            if st.button(application.TITLE):
```
Comment (SimonBiggs) [inline lib/pymedphys/_streamlit/index.py:148]: Here

File: lib/pymedphys/_streamlit/index.py [hunk 3]
```diff
@@ -194,6 +162,7 @@ def main():
         swap_app("index")
 
     if session_state.app != "index":
+        st.title(APPLICATION_OPTIONS[session_state.app].TITLE)
```
Comment (SimonBiggs) [inline lib/pymedphys/_streamlit/index.py:165]: and here

**Review by SimonBiggs — COMMENTED** [review]
> Are CATEGORY and TITLE special variables known to streamlit?
It isn't clear how these are getting used (I didn't see them in the files with calls to st.title(...))

They're not known to Streamlit, I'm calling them at the following locations:

**Pull request merged**